portfolio/src/main/webapp/js/script.js

```diff
@@ -18,23 +18,92 @@
 */
 filterGallery('all');
 function filterGallery(galleryFilter) {
-    var documents = document.getElementsByClassName("gallery");
-    
-    if (galleryFilter == "all")
-        galleryFilter = "";
-    
-    for (var index = 0; index < documents.length; index++) {
-        documents[index].className = documents[index].className.replace("show-gallery-img", "");
+    let documents = document.getElementsByClassName('gallery');
+
+    if (galleryFilter == 'all') galleryFilter = '';
+
+    for (let index = 0; index < documents.length; index++) {
+        documents[index].className = documents[index].className.replace(
+            'show-gallery-img',
+            ''
+        );
         if (documents[index].className.includes(galleryFilter)) {
-            documents[index].className += " show-gallery-img"
+            documents[index].className += ' show-gallery-img';
```

Comment:
Instead of appending the class name, jQuery actually has an `addClass()` function!

Check it out: https://api.jquery.com/addClass/